apollo-router/tests/integration/telemetry/otlp.rs

```diff
@@ -239,7 +239,7 @@ async fn test_otlp_request_with_zipkin_trace_context_propagator_with_datadog(
         .await?;
     // ----------------------
     // Testing for a trace containing a tracestate m and psr with psr set to 1 for DD and an unsampled trace, so it should be sent to the otlp exporter with sampling priority set to 1
-    // But it should send the trace to subgraph as the tracestate contains a psr set to 1
+    // It should not send the trace to the subgraph as we didn't use the datadog propagator and therefore the trace will remain unsampled.
```

Comment:
@bnjjj This is a key test change.

Reply:
Are we sure we should not take care of this tracestate ? Do we have any specs or docs to help us to choose ?

Reply:
https://github.com/DataDog/dd-trace-js/blob/fcd3ab8244800e64b01bdd746933812dada90ad5/packages/dd-trace/test/opentelemetry/span_context.spec.js#L58